src/functionaltests/java/com/ericsson/ei/notifications/trigger/SubscriptionNotificationSteps.java

```diff
@@ -187,6 +195,14 @@ public void send_one_previous_event() throws Throwable {
         eventManager.sendEiffelEvents(EIFFEL_EVENTS_JSON_PATH, eventNamesToSend);
     }
 
+    @Then("^Default values mail notification are added$")
+    public void default_values_mail_notification() {
+        if (emailSender.getSender() != null && emailSender.getSubject() != null) {
```

Comment:
This if statement is not needed, sin we always want to check the assert. If the value is null then the assertion goes wrong either way.

Reply:
okay You mean this scenario is not required? shall I remove it? As I have also added the subject assert in mail_subscriptions_were_triggered method.

Reply:
No the scenario should still be there, but you can remove line 200: "if (emailSender.getSender() != null && emailSender.getSubject() != null)" and always check the assert on these values.

Reply:
okay